### DISCUSSION TITLE
Add Webhook support for Calendar events (Creating, modifying, accepting, ect..)

### DISCUSSION THREAD

**Discussion by melizalde-ds** [sreeise/graph-rs-sdk#501] [Ideas]
Hi!
I am using this to create a personal project to learn more about Rust. I am doing a web-based application that requires the use of webhooks. I haven't found anything about subscribing to a webhook to receive notifications when an event is added, changed, etc. 
I think adding support to webhooks could be something helpful for many people. I would love also to help as much as I can. Thanks!

**Comment by sreeise** [comment]
Apologies I am getting back to this so late. My day job has been very busy in the past 6 months and I had very little time. 

Yeah I could def see this being something that is helpful, but im not 100% its appropriate for this crate. The webhook would require an HTTP endpoint correct? This crate is really only here to provide API integration and not necessarily host an actual API. And im not sure doing so for just webhooks is worth the effort. Im open to ideas but just from what I understand about it now I don't think it makes sense for this crate.

**Reply by melizalde-ds** [reply]
Hey! No worries, I understand entirely. Thanks!

**Reply by melizalde-ds** [reply]
Hey! No worries, I understand entirely. Thanks!
